test/e2e/fixtures/google-pay.js

```diff
@@ -33,6 +33,7 @@ class GooglePayElementComponent extends Component {
             <h2>{this.props.title}</h2>
             <h3>FramePay version: {this.props.Rebilly.version}</h3>
             <div className="flex-wrapper">
+                {prettyDebugRender(this.state)}
```

Comment:
I think a couple of these debug prints should also be removed?

Reply:
Thank you for the comment! `prettyDebugRender` is used across all other files ot provide just visual identation of state. I added it to Google Pay and PayPal to keep consistent with the rest of renderings. 

It is not used right now for PayPal and Google Pay so it mostly gives us the same visual layout and chance to scale to test the tokenReady event.

Reply:
Ah nice one :) thanks